openslides/users/static/js/users/site.js

```diff
@@ -912,6 +907,9 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
                     // Success: User logged in.
                     operator.setUser(response.data.user_id);
                     $scope.closeThisDialog();
+                    setTimeout(function(){
```

Comment:
Proposal: Close the dialog after the 1000 ms waiting time, i. e. move this call into the timeout.